models/base_sources/dune_upload_sources.yml

```diff
@@ -5,4 +5,7 @@ sources:
     description: >
       OP chain metadata uploaded by the OPLabs team.
     tables:
-      - name: op_stack_chain_metadata
\ No newline at end of file
+      - name: op_stack_chain_metadata
+      - name: op_airdrop1_addresses_detailed_list
+      - name: op_airdrop2_addresses_detailed_list
+      - name: op_airdrop_3_addresses_detailed_list
```

Comment:
Tricky for `op_airdrop_3_addresses_detailed_list`

Reply:
fyi @MSilb7 
airdrop 3 has a different naming standard, it appears, with an extra `_` in there

i won't consider a blocker here, but if you ever fix, make sure to update sources file and all source usage in models downstream